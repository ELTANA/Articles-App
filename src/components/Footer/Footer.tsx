const Footer = () => {
  return (
    <footer className="py-4 mt-3 border-t-2 border-t-blue-700">
      <p className="text-center">© 2024 Articles App — Lotana</p>
    </footer>
  );
};

export default Footer;
