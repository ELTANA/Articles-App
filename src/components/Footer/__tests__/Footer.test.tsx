import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Footer from '../Footer';

describe('Footer component', () => {
  test('renders footer with the correct text', () => {
    render(<Footer />);
    const footerElement = screen.getByText(/© 2024 Articles App — Lotanna Kyrian/i);
    expect(footerElement).toBeInTheDocument();
  });

  test('matches snapshot', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
