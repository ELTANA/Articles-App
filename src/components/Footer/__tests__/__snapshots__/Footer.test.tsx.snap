// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="py-4 mt-3 border-t-2 border-t-blue-700"
  >
    <p
      class="text-center"
    >
      © 2024 Articles App — Lotana
    </p>
  </footer>
</DocumentFragment>
`;
